packages/jest-core/src/runJest.ts

```diff
@@ -167,6 +168,28 @@ export default async function runJest({
   const sequencer = new Sequencer({contexts, globalConfig});
   let allTests: Array<Test> = [];
 
+  // Initialize Vite integration for each context if enabled
+  performance.mark('jest/viteSetup:start');
+  const viteServers = await Promise.all(
```

Comment:
Can we have 1 single global Vite server? Why do we need to create multiple servers like this? Can you investigate how Vitest does?